src/grpc/iot_config/hpr_route_stream_worker.erl

```diff
@@ -253,39 +313,46 @@ maybe_init_from_file(Path) ->
             ok = file:write_file(Path, erlang:term_to_binary(#{}));
         {error, _Reason} ->
             lager:error("failed to open backup file (~s) ~p", [Path, _Reason]);
-        {ok, Map} ->
+        {ok, {RoutesMap, EUIPairs, DevAddrRanges}} ->
             maps:foreach(
                 fun(_ID, Route) ->
-                    hpr_route_ets:insert(Route)
+                    hpr_route_ets:insert_route(Route)
                 end,
-                Map
-            )
+                RoutesMap
+            ),
+            lists:foreach(fun hpr_route_ets:insert_eui_pair/1, EUIPairs),
+            lists:foreach(fun hpr_route_ets:insert_devaddr_range/1, DevAddrRanges)
     end.
 
--spec open_backup_file(Path :: string()) -> {ok, map()} | {error, any()}.
+-spec open_backup_file(Path :: string()) -> {ok, {map(), list(), list()}} | {error, any()}.
```

Comment:
I'm wondering if there's still a benefit of having Routes in a map. 
We can `lists:usort` and keep everything as a list?